dictionary_app/app.py

```diff
@@ -0,0 +1,89 @@
+import pymongo
+from flask import Flask, render_template, request, redirect, jsonify
+import requests
+
+client = pymongo.MongoClient("mongodb+srv://base_user:base_user_password@cluster0.dbcb9.mongodb.net/first")
+db = client.first
+collectiondictionary = db.gokberk
+
+
+app = Flask(__name__)
+
+if __name__ == '__main__':
+    app.run()
+
+
+@app.route('/dictionary')
+def home_dictionary():
+    return render_template('search.html')
+
+
+@app.route("/dictionary-search-synonym/<word>", methods=["GET"])
+def synonym_dictionary(word):
+    url_synonym = "https://wordsapiv1.p.rapidapi.com/words/" + word + "/synonyms"
+    headers_synonym = {
+        'x-rapidapi-key': "4e9bda707amshaa2a2e7a0c08b4dp1f1264jsn3bb1fb127723",
+        'x-rapidapi-host': "wordsapiv1.p.rapidapi.com"
+    }
+    response_synonym = requests.request("GET", url_synonym, headers=headers_synonym)
+    st = response_synonym.json()
+    syns = st["synonyms"]
+    synsdict = {}
+    for i, val in enumerate(syns):
+        synsdict[i+1] = val
+    return synsdict
+
+
+@app.route("/dictionary-search-definition/<word>", methods=["GET"])
+def definition_dictionary(word):
+    url_definition = "https://wordsapiv1.p.rapidapi.com/words/" + word + "/definitions"
+    headers_definition = {
+        'x-rapidapi-key': "4e9bda707amshaa2a2e7a0c08b4dp1f1264jsn3bb1fb127723",
```

Comment:
you should not put your key in the code. You can put it on the database and access it in here